kcp/setup-kcp-cps.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+set -ex
+
+CPS_KUBECONFIG="${CPS_KUBECONFIG:-cps-kubeconfig}"
+WORKLOAD_KUBECONFIG="${WORKLOAD_KUBECONFIG:-$HOME/.kube/config}"
+WORKSPACE="gitops-service"
+SYNCER_IMAGE="${SYNCER_IMAGE:-ghcr.io/kcp-dev/kcp/syncer:v0.5.0-alpha.1}"
```

Comment:
@chetan-rns, on a quick look, this seems to be a little odd, don't we need to have the same syncer image for the same kcp release?

Reply:
I think you are looking at an older commit. I've updated the image tag to v0.7.1 https://github.com/redhat-appstudio/managed-gitops/blob/dc9fed4eaadbfc04ea6c8147c261e8ef8d874fae/kcp/setup-kcp-cps.sh#L8

Reply:
Aha, my bad!